pglast/printers/ddl.py

```diff
@@ -1972,11 +2182,17 @@ def grant_stmt(node, output):
     else:
         output.write('REVOKE ')
         preposition = 'FROM'
+    if not node.is_grant and node.grant_option:
+        output.write('GRANT OPTION FOR ')
     if node.privileges:
         output.print_list(node.privileges)
     else:
         output.write('ALL PRIVILEGES')
-    object_name = OBJECT_NAMES[node.objtype.value]
+    # hack for OBJECT_FOREIGN_SERVER
+    if node.objtype.value == 17:
```

Comment:
Can you use `enums.ObjectType.OBJECT_FOREIGN_SERVER` instead of hardcoded number?